README.md

```diff
@@ -0,0 +1,43 @@
+# `go-depsync`
+
+`go-depsync` is a small command that identifies common dependencies with a given package, referred to as parent, and outputs a `go get` command that allows aligning the version of common dependencies with those of the parent package.
+
+It is useful for extensions and plugins that are built together with a core package, and whose dependencies need to be compatible.
+
+## Usage
+
+`xk6-depsync` can be installed using `go install`:
+
+```console
+$ go install github.com/grafana/go-depsync
+```
+
+After that, it is ready to be used:
+
+```bash
+go-depsync --parent go.k6.io/k6
+```
+
+If the `go.mod` file for the local package is not on the working directory, a path to it can also be specified:
+
+```bash
+go-depsync --gomod /path/to/go.mod --parent go.k6.io/k6
+```
+
+`go-depsync` produces an output similar to the following:
+
+```console
+$ go-depsync --parent=go.k6.io/k6
+2023/11/17 12:59:01 Found parent go.k6.io/k6@v0.46.0
+2023/11/17 12:59:01 Mismatched versions for github.com/spf13/afero: v1.2.2 (this package) -> v1.1.2 (parent)
+2023/11/17 12:59:01 Mismatched versions for golang.org/x/sys: v0.11.0 (this package) -> v0.9.0 (parent)
+2023/11/17 12:59:01 Mismatched versions for github.com/spf13/cobra: v1.5.0 (this package) -> v1.4.0 (parent)
+2023/11/17 12:59:01 Mismatched versions for google.golang.org/grpc: v1.57.0 (this package) -> v1.56.1 (parent)
+go get github.com/spf13/afero@v1.1.2 golang.org/x/sys@v0.9.0 github.com/spf13/cobra@v1.4.0 google.golang.org/grpc@v1.56.1
```

Comment:
Could we have a table-based output? Something like https://github.com/rodaine/table where we will have something like the following.

Parent  | Current | Package

v0.1.0    v0.2.0      mypackage.com/blah

In any case, I don't think we need the timestamp

Reply:
Good idea, tables are always nice. In the past I've used https://github.com/olekukonko/tablewriter, but I'm fine with either. Do you suggest the repo above for any particular reason?

Reply:
Not really, pick whatever is convenient in your opinion. If you have already some knowledge around one specific then do with it. We can always refactor it in the future if it is required.

Reply:
In the past, I've only used the std lib as https://pkg.go.dev/text/tabwriter

Reply:
Okay, gave this a swing with the one I mentioned above (just for familiarity). It should be quite easy to swap it for another if we like it more :)